src/Logging/CriticalErrorProcessor.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Common\Logging;
+
+use Linio\Common\Exception\CriticalError;
+
+class CriticalErrorProcessor
+{
+    public function __invoke(array $record): array
+    {
+        $exception = $record['context']['exception'] ?? null;
+
+        if (!$exception instanceof CriticalError) {
+            return $record;
+        }
+
+        // Monolog doesn't use psr\log\LogLevel values internally
```

Comment:
I knew about the getLevelName, but didn't about the toMonologLevel, so because of that, I decided to just hardcode it. I will make the change.